docs/developer/index.rst

```diff
@@ -12,6 +12,7 @@ Developer
     using_omnisci
     using_pyarrow_on_ray
     using_sql_on_ray
+    using_pandas_on_python
```

Comment:
Let's put this next to `using_pandas_on_ray` and `using_pandas_on_dask`.

Reply:
Now this is placed under `pandas on Dask`